server/src/main/java/org/apache/iotdb/db/qp/physical/crud/QueryPlan.java

```diff
@@ -36,6 +36,8 @@
   private int rowLimit = 0;
   private int rowOffset = 0;
 
+  private int level = -1;
```

Comment:
I think this `level` parameter shouldn't be inserted here in Queryplan which is the base class. Maybe groupByTimePlan or AggregationPlan is better.